docs/type_annotations.md

```diff
@@ -102,6 +102,7 @@ b : dict[int,bool] = {1: False, 4: True}
 c : dict[int,complex] = {}
 ```
 Strings are not yet supported as keys in Fortran.
```

Comment:
Supporting strings as keys in Fortran sounds like a very important task, but I could not find an issue about it.

Reply:
Hmm. You're right :thinking: I'll open one

Reply:
https://github.com/pyccel/pyccel/issues/2278